Libraries/LibPthread/pthread.h

```diff
@@ -35,7 +35,7 @@
 __BEGIN_DECLS
 
 int pthread_create(pthread_t*, pthread_attr_t*, void* (*)(void*), void*);
-void pthread_exit(void*);
+[[noreturn]] void pthread_exit(void*);
```

Comment:
Nope, this is a C header.

Reply:
My bad. Should revert pthread.h

Reply:
Nah, it's fine, just use the C syntax for attributes

Reply:
actually, its only extern "C" declarations. the actuall code is in pthread.cpp and is compiled with c++ compiler.
Why we should prefer __attribute__ ((noreturn)) ?

Reply:
Because it's a C header: pure C code (mainly from ports of third-party software) can `#include` it.

Reply:
Of course! thanks.